supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesBackendLifecycleActorFactory.scala

```diff
@@ -101,7 +101,7 @@ object JesBackendLifecycleActorFactory {
       RuntimeAttributeDefinition(FailOnStderrKey, Option(staticDefaults(FailOnStderrKey)), usedInCallCaching = true),
       RuntimeAttributeDefinition(MemoryKey, Option(staticDefaults(MemoryKey)), usedInCallCaching = false),
       RuntimeAttributeDefinition(DisksKey, Option(staticDefaults(DisksKey)), usedInCallCaching = false),
-      RuntimeAttributeDefinition(ZonesKey, Option(staticDefaults(ZonesKey)), usedInCallCaching = false),
+      RuntimeAttributeDefinition(ZonesKey, None, usedInCallCaching = false),
```

Comment:
Ok so the issue here is that we *could* reach into the config and grab the value that we want directly in `JesRuntimeAttributes` which would actually make the code on this PR a lot more streamlined (to the point @kcibul made about how many files were touched). However that violates the model that everything else is using in the JES backend which is that `JesAttributes` is passed the config for the JES backend and it is deconstructing everything and passing it along. In this case centralization of logic causes the issue.

I'm going to leave as-is and leave it as another point in favor of my stance that runtime attrs need to be rethought in general